e2e/provision/Vagrantfile

```diff
@@ -20,6 +20,9 @@ Vagrant.configure('2') do |config|
   config.vm.box_check_update = false
   config.vm.synced_folder './', '/vagrant'
 
+  config.vm.network 'forwarded_port', guest: 7007, guest_ip: '127.0.0.1', host: 7007
```

Comment:
nice, so in the vagrant case you do not need the -L in your ssh? We should document that. You will still need to do the kubectl port forwarding, unless we expose the UI on a 172.18.0.0/20 address. Which I think in general would be fine for the sandbox, even though there is no authentication on the UI in sandbox mode. I'll create a separate issue to discuss that.